modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -479,6 +479,11 @@ properties:
             description: |
               If set to `true`, Sync Gateway will use views instead of GSI for system functions like authentication and replication.
             default: 'false'
+          view_query_timeout_secs:
+            type: integer
+            description: |
+              The view query timeout in seconds. This property allows you to specify the time Sync Gateway should wait for a view query response from Couchbase Server before it times out. The value is ignored when the `databases.$db.use_views` property is set to `true`.
+            default: 75
```

Comment:
This timeout will be used for either view or N1QL queries, so the last sentence can be omitted (or modified to clarify that).